### PR TITLE
Feature/165-Add navigation link and integrate with Resumes List

### PR DESCRIPTION
**Pull request by ela-ocana-jalafund**
[AB#165](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/165)
https://dev.azure.com/funjaladev/DevLevel3/_boards/board/t/DevLevel3%20Team/Stories/?workitem=165

# Overview
- Add navigations buttons with links routes into Resumes list. (Navigations between resumes list, create resume and resume detail)
- Add a button "New Resume" in resumes list view to create a new Resume.

## Outstanding Tasks
- Add buttons into boxes of resumes list. Each Resume Box has a button to see Resume's detail. This task is added with US #38
because This task need changes with the US #38 .
## Screenshots
imagen

### CODE REVIEW

**Review by niver-copa-jalafund — DISMISSED** [review]

**Review by DaleneyRuben — CHANGES_REQUESTED** [review]
@ela-ocana-jalafund Please fulfill the pr template, please add a description for this PR.

**Review by niver-copa-jalafund — DISMISSED** [review]

**Comment by DaleneyRuben** [comment]
@ela-ocana-jalafund I can see this is related with [this](https://github.com/fundev3/teamup-webapp/pull/28) pr. We need to wait until that pr gets merged. Unfortunately this work depends on another story to be completed.

**Comment by DaleneyRuben** [comment]
@niver-copa-jalafund please take your time to review the PRs. You're just approving anything.

**Comment by DaleneyRuben** [comment]
@ela-ocana-jalafund The dependent PR is already merged in develop please merge your branch with develop.

**Comment by jorge-lema-jalafund** [comment]
@ela-ocana-jalafund  your PR has merge conflicts. Solve them and then merge the PR

**Review by lilian-delgado-jalafund — DISMISSED** [review]
Looks good to me, however I will suggest you to use margin: 0px 50px; so you will save some lines instead of making margin-left:50px; & margin-right:50px; It works for padding too. Save it for next time.

**Comment by martin-callejas-jalafund** [comment]
Thanks for adding a screenshot, it's so informative for the reviewer

File: src/components/resumes/Details.jsx
Comment (gustavo-rojas-jalafund) [inline src/components/resumes/Details.jsx]: Please remove this coment

Comment (ela-ocana-jalafund) [reply src/components/resumes/Details.jsx]: Done

**Comment by ela-ocana-jalafund** [comment]
> 
> 
> Looks good to me, however I will suggest you to use margin: 0px 50px; so you will save some lines instead of making margin-left:50px; & margin-right:50px; It works for padding too. Save it for next time.

**Review by gustavo-rojas-jalafund — DISMISSED** [review]

**Review by niver-copa-jalafund — DISMISSED** [review]
I review your changes, in my perseption is good

**Review by lilian-delgado-jalafund — DISMISSED** [review]
Lgtm

File: src/components/resumes/Resumes.jsx
Comment (jorge-lema-jalafund) [inline src/components/resumes/Resumes.jsx]: I think we are already using lower case for our css classes. 
e.g.  resumes-container instead of Resumes-Container
Please re-write them.

**Review by jorge-lema-jalafund — CHANGES_REQUESTED** [review]

File: src/components/resumes/Resumes.jsx
Comment (ela-ocana-jalafund) [reply src/components/resumes/Resumes.jsx]: ohh noo!! ok I'll fix.

**Review by niver-copa-jalafund — DISMISSED** [review]
I was review your code, is fine for me

**Review by lilian-delgado-jalafund — DISMISSED** [review]

**Review by jorge-lema-jalafund — DISMISSED** [review]
lgtm

File: src/components/resumes/Box.jsx
Comment (DaleneyRuben) [inline src/components/resumes/Box.jsx]: NABD you can use template literals here

File: src/components/resumes/Box.jsx
Comment (DaleneyRuben) [inline src/components/resumes/Box.jsx]: NABD you can use template literals here

File: src/components/resumes/Details.jsx
Comment (DaleneyRuben) [inline src/components/resumes/Details.jsx:5]: this function does not exist on ResumesAPI

**Review by DaleneyRuben — CHANGES_REQUESTED** [review]

File: src/components/resumes/ResumesAPI.js
Comment (DaleneyRuben) [inline src/components/resumes/ResumesAPI.js]: why are you changing this file?

**Review by niver-copa-jalafund — DISMISSED** [review]
This looks fine

**Review by lilian-delgado-jalafund — DISMISSED** [review]
lgtm

**Review by niver-copa-jalafund — APPROVED** [review]

**Pull request closed without merging**
